torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -2122,6 +2127,19 @@ def _pop_config_kwargs(config: dict[str, Any]) -> dict[str, Any]:
     return popped
 
 
+def config_to_dict(config: Config) -> dict[str, Any]:
+    return {
+        **config.kwargs,
+        "num_warps": config.num_warps,
+        "num_stages": config.num_stages,
```

Comment:
what happened to the other few fields from Config that are handled by `_pop_config_kwargs`?  (`num_ctas`, `maxnreg`)?

Reply:
I excluded them since it will break BC with older versions of Triton.  We should add them at some point, but for a refactor PR I'm trying to limit changes.